pkg/k8s/k8s.go

```diff
@@ -38,6 +38,7 @@ type DeploymentData struct {
 	ReleaseName    string
 }
 
+// TODO: seems like this struct is not needed
 type DataFromK8sAPI struct {
 	KubeData       *KubeData
 	DeploymentData *DeploymentData
```

Comment:
@alavrovinfb maybe I should remove this struct also? It seems we aren't gonna need it.

Reply:
It seems, the struct isn't used anywhere and should be removed, but please double-check it.